example/src/components/ButtonContainer/ButtonContainer.jsx

```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import './ButtonContainer.css'
 
+/**
+ * One custom component from a application that uses Nice!, like Search!
```

Comment:
I'm not sure what this means

Reply:
I agree. But that component might change in `creating-example-app` branch, so should add a comment about it there.